### ISSUE TITLE
add the latest trait data

### ISSUE THREAD

**Issue by aammd** [SrivastavaLab/bwgtools#31]
the `forupload` file contains our most recent bwgdb insect trait data. These should be combined with this package. add these data to the package, and get [get_bwg_tools](https://github.com/SrivastavaLab/bwgtools/blob/06114b9c738a589355c1ea673fd7143fb130351b/R/reading.R#L105)

**Comment by dsrivast** [comment]
@aammd : Has this been done?

**Comment by aammd** [comment]
No, this is going to need a redesign. I have some time in the next airport; could probably fix it then!

**Comment by dsrivast** [comment]
Thanks, because it is holding things up a bit my end...

Diane

-- 
Diane Srivastava
Professor
Dept of Zoology & Biodiversity Research Centre,
University of British Columbia
6270 University Blvd
Vancouver BC Canada V6T 1Z4
604-822-1350 (no voicemail)
